pkgs/development/compilers/unison/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
   dontConfigure = true;
 
   nativeBuildInputs = stdenv.lib.optional (!stdenv.isDarwin) autoPatchelfHook;
-  buildInputs = stdenv.lib.optionals (!stdenv.isDarwin) [ ncurses5 zlib gmp ];
+  buildInputs = [ less ] ++ (stdenv.lib.optionals (!stdenv.isDarwin) [ ncurses5 zlib gmp ]);
```

Comment:
Does it really link against less?

Reply:
no, but it uses it at runtime, and this is the way i know to make it available in a pure shell.

Better approaches welcomed :)

Reply:
makeWrapper would be a good fit here.

Reply:
Sorry, I don't follow.  A quick glance at both [`make_wrapper.sh`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/make-wrapper.sh) and [docs](https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-functions) doesn't really help me understand what you mean.  Do you have an example you can point me to?

Reply:
No problem: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/somafm-cli/default.nix#L24

Reply:
Ah great, thanks!  Updated and tested that it works correctly in a pure shell.